src/tests/unit/audit_remediation/test_audit_remediation_base.py

```diff
@@ -309,6 +309,17 @@ def test_result_query_export_file(cbcsdk_mock):
         os.remove(tempfile)
 
 
+def test_result_query_export_lines(cbcsdk_mock):
+    """Tests exporting the results of a query as a list of lines."""
+    input = "AAA\r\nBBB\r\nCCC"
+    cbcsdk_mock.mock_request("POST_LINES", "/livequery/v1/orgs/test/runs/run_id/results/_search?format=csv",
+                             CBCSDKMock.StubResponse(input, 200, input, False))
+    api = cbcsdk_mock.api
+    result_query = api.select(Result).run_id("run_id")
+    output = result_query.export_csv_as_lines()
```

Comment:
I thought this was suppose to return an iterator not a list?

Reply:
Perhaps I misspoke. It returns, not an _iterator,_ but an _iterable._  I can fix the docstrings to reflect that.

Reply:
Have you run this against the CBC APIs? I'm curious if the mocking matches the behavior of a live run?

Reply:
To verify my understanding, this won't read the entire file into memory instead it will process line by line as requested. If we load it into a list that would but it could be processed in a for loop to prevent that.

Reply:
Oof, you're right.  I need to encapsulate the output in a list to match the equivalent test that's made at the low level. Doing so.

And yes, `export_csv_as_lines()` _has_ been run against a live CBC server. The test script I used is up in the PR description.